third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -90,9 +91,38 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 
 									"versioned_expr": {
 										Type:         schema.TypeString,
-										Required:     true,
+										Optional:     true,
 										ValidateFunc: validation.StringInSlice([]string{"SRC_IPS_V1"}, false),
 									},
+
+<% unless version == 'ga' -%>
+									"expr": {
+										Type:     schema.TypeList,
+										Optional: true,
+										MaxItems: 1,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"expression": {
+													Type:     schema.TypeString,
+													Required: true,
+												},
+												// These fields are not yet supported (Issue #4497: mbang)
```

Comment:
nit: in GH this will link to TPGB since this block doesn't exist in TPG. I think saying terraform-providers/terraform-provider-google#4497 will work.